jdi-light-html-tests/pom.xml

```diff
@@ -42,7 +42,7 @@
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>7.0.0</version>
+            <version>6.14.3</version>
```

Comment:
7.0.0

Reply:
Was changed in master, probably it shouldn't be fixed